src/plugins/1/essentials/getting-started/index.md

```diff
@@ -34,87 +20,28 @@ git clone https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate
 cd kuzzle-core-plugin-boilerplate
 docker-compose -f docker/docker-compose.yml up
 // Kuzzle stack with the plugin is ready
-// Edit the file lib/index.js, 
+// Edit the file lib/index.js,
 // the Kuzzle stack will automaticaly restart to include your modifications
 ```
 
-## Prerequisites
-
-### Location
-
-Plugins are subdirectories that must be put at the following location: `<kuzzle_install_dir>/plugins/enabled`.
+The provided `docker-compose` launches a Kuzzle stack with the `pm2` module, you can:
 
-The recommended way to install a plugin is to put it in `plugins/available`, and then link it to the `plugins/enabled` directory.
+- Enable hot reload by watching the plugin's files.
+- Configure it using the file next to the `docker-compose` inside the docker folder.
 
-### Node.js modules
+The main Plugin class is defined in the `index.js`. You can start edit it adding:
 
-Kuzzle loads plugins as [Node.js modules](https://nodejs.org/dist/latest-v8.x/docs/api/modules.html).
+- [Hooks]({{ site_base_path }}plugins/1/essentials/hooks/)
+- [Pipes]({{ site_base_path }}plugins/1/essentials/pipes/)
+- [Controllers]({{ site_base_path }}plugins/1/essentials/controllers/)
+- [Authentication Strategies]({{ site_base_path }}plugins/1/strategies/overview/)
 
-This means that a plugin directory must contain either:
-
-* an `index.js` file 
-
-and/or:
-
-* a valid [package.json](https://docs.npmjs.com/files/package.json) file. If the plugin's entrypoint is not the root `index.js` file, then the [main](https://docs.npmjs.com/files/package.json#main) property must be filled
-
-### manifest.json
-
-Kuzzle needs a few information to make your plugin work properly. These information must be provided in a `manifest.json` file, in the plugin directory.
-
-The following properties can be defined in this `manifest.json` file:
-
-* `name` (**required**): plugin unique identifier. Names can only contain lowercase letters, numbers, hyphens and underscores. 
-* `kuzzleVersion`: a non-empty string describing a [semver range](https://www.npmjs.com/package/semver#ranges), limiting the range of Kuzzle versions supported by this plugin. If not set, a warning is displayed on the console, and Kuzzle assumes that the plugin is only compatible with Kuzzle v1.x
-
-{{{deprecated "1.5.0"}}} Kuzzle still allows plugins to be loaded without a <code>manifest.json</code> file, for backward compatibility reasons, falling back to the <a href=https://docs.npmjs.com/files/package.json#name>package.json</a> file to retrieve the plugin's name. This will change in next major releases of Kuzzle.
-
----
+Since the `init` function is the very first to be called, it's loading the `configuration` and the `context`.
```

Comment:
Inverted causes and consequences: it's because we need to provide the configuration and a context to plugins that we ask those to expose an `init` function.

Also, the init function does not load anything, it merely stores what we give to it.